content/director-blobstore-signed-urls.md

```diff
@@ -72,6 +74,15 @@ For DAV blobstores, please also configure:
 
 * `blobstore.secret`: The secret used to calculate the signed urls' signature
 
+### Removing blobstore credentials for agent
```

Comment:
I was actually thinking of a note why `blobstore.director.user` and `blobstore.director.password` still need to be defined, which could confuse people.

Reply:
The director still always need to write blobs when BOSH Releases are uploaded to the Director. Is that what you mean @beyhan?

Reply:
@bgandon I mean this part of the discussion in Slack: https://cloudfoundry.slack.com/archives/C02HPPYQ2/p1632395452103000?thread_ts=1632389157.101900&cid=C02HPPYQ2

Reply:
Correct me if I'm wrong but it's different. Here we're talking about provisioning users on the blobstore. In Slack, we were concerned about providing credentials through the CPI (and the `create_vm` method) so that the agent is not confused with missing credentials for accessing the blobstore.

The concern about missing credentials finally comes down to patching the https://github.com/cloudfoundry/bosh-davcli, as currently investigated in our internal User Story.

Here we should not address this, so I think mentioning  `blobstore.director.user` and `blobstore.director.password` is just about re-specifying that the Director still needs to write blobs when BOSH Releases are uploaded on it.